packages/manager/src/features/Firewalls/FirewallDetail/Rules/FirewallRuleTable.tsx

```diff
@@ -275,88 +305,101 @@ const FirewallRuleTableRow = React.memo((props: FirewallRuleTableRowProps) => {
 
   const actionMenuProps = {
     disabled: status === 'PENDING_DELETION' || disabled,
-    idx: id,
+    idx: index,
     triggerCloneFirewallRule,
     triggerDeleteFirewallRule,
     triggerOpenRuleDrawerForEditing,
   };
 
+  const {
+    active,
+    attributes,
+    isDragging,
+    listeners,
+    setNodeRef,
+    transform,
+    transition,
+  } = useSortable({ id });
+
+  const isActive = Boolean(active);
+
+  // dnd-kit styles
+  const rowStyles = {
+    '& td': {
+      // Highly recommend to set the `touch-action: none` for all the draggable elements-
+      // in order to prevent scrolling on mobile devices.
+      // refer to https://docs.dndkit.com/api-documentation/sensors/pointer#touch-action
+      touchAction: 'none',
+    },
```

Comment:
I'm not sure if this is recommended from an accessibility perspective, but I'm finding that it is helpful to make it more obvious that a row is focused

```suggestion
    },
    ':focus': {
      backgroundColor: 'lightblue',
    },
```

Reply:
`lightblue` color looks a bit odd in dark mode and makes the row text unreadable. Is there a more theme-sensitive background color we could use for focus, other than the one we used for edit, clone, and delete statuses (to avoid color conflict)?
![Screenshot 2024-11-11 at 3 32 15 PM](https://github.com/user-attachments/assets/de54e898-3f25-43eb-90e5-ea8ab6a3bc8e)